utils/kubernetes/kube_state_processor.py

```diff
@@ -228,38 +286,70 @@ def kube_pod_status_phase(self, message, **kwargs):
             status = self.pod_phase_to_status.get(phase, self.kube_check.UNKNOWN)
             self.service_check(check_basename + phase, status, tags=tags)
 
-    def kube_node_status_ready(self, message, **kwargs):
-        """ The ready status of a cluster node. """
-        service_check_name = NAMESPACE + '.node.ready'
+    # TODO: implement kube_pod_status_ready
```

Comment:
Maybe refer to https://github.com/DataDog/dd-agent/issues/3045 if you planned to keep that open, or open one or more issues for this so you or others can pick it up.